inc/Engine/Admin/Settings/Page.php

```diff
@@ -673,78 +673,85 @@ private function assets_section() {
-					'container_class'   => [
+				'optimize_css_delivery_method' => [
+					'type'                    => 'radio_buttons',
+					'label'                   => __( 'Optimize CSS delivery', 'rocket' ),
+					'container_class'         => [
 						'wpr-field--children',
-					],
-					'placeholder'       => "/wp-content/plugins/some-plugin/(.*).css\n.css-class\n#css_id\ntag",
-					'parent'            => 'remove_unused_css',
-					'section'           => 'css',
-					'page'              => 'file_optimization',
-					'default'           => [],
-					'sanitize_callback' => 'sanitize_textarea',
-					'input_attr'        => [
-						'disabled' => get_rocket_option( 'remove_unused_css' ) ? 0 : 1,
+						'wpr-field--optimize-css-delivery ',
+					],
+					'buttons_container_class' => '',
+					'parent'                  => 'optimize_css_delivery',
+					'section'                 => 'css',
+					'page'                    => 'file_optimization',
+					'default'                 => 'remove_unused_css',
+					'sanitize_callback'       => 'sanitize_checkbox',
+					'options'                 => [
+						'remove_unused_css' => [
+							'label'       => __( 'Remove Unused CSS', 'rocket' ),
+							// translators: %1$s = opening <a> tag, %2$s = closing </a> tag.
+							'description' => sprintf( __( 'Removes unused CSS per page and helps to reduce page size and HTTP requests for better performance. %1$sMore info%2$s', 'rocket' ), '<a href="' . esc_url( $rucss_beacon['url'] ) . '" data-beacon-article="' . esc_attr( $rucss_beacon['id'] ) . '" target="_blank">', '</a>' ),
+							'sub_fields'  => [
+								'remove_unused_css_safelist' =>
+								[
+									'type'              => 'textarea',
+									'label'             => __( 'CSS safelist', 'rocket' ),
+									'description'       => __( 'Specify CSS filenames, IDs or classes that should not be removed (one per line).', 'rocket' ),
+									'placeholder'       => "/wp-content/plugins/some-plugin/(.*).css\n.css-class\n#css_id\ntag",
+									'default'           => [],
+									'value'             => '',
+									'sanitize_callback' => 'sanitize_textarea',
+									'parent'            => '',
+									'section'           => 'css',
+									'page'              => 'file_optimization',
+									'input_attr'        => [
+										'disabled' => get_rocket_option( 'remove_unused_css' ) ? 0 : 1,
+									],
+								],
+							],
+						],
+						'async_css'         => [
+							'label'       => __( 'Load Asynchronously', 'rocket' ),
+							'description' => is_plugin_active( 'wp-criticalcss/wp-criticalcss.php' ) ?
+								// translators: %1$s = plugin name.
+								sprintf( _x( 'Optimize CSS Delivery is currently handled by the %1$s plugin. If you want to use WP Rocket’s Optimize CSS Delivery option, disable the %1$s plugin.', 'WP Critical CSS compatibility', 'rocket' ), 'WP Critical CSS' ) :
```

Comment:
```suggestion
								sprintf( _x( 'Critical path CSS is currently handled by the %1$s plugin. If you want to use WP Rocket’s Optimize CSS Delivery option, disable the %1$s plugin.', 'WP Critical CSS compatibility', 'rocket' ), 'WP Critical CSS' ) :
```

Reply:
I think it should be "Loads CSS asynchronously" like the button